hypertrace-graphql-label-application-rules-schema/src/main/java/org/hypertrace/graphql/label/application/rules/schema/shared/Action.java

```diff
@@ -51,6 +51,6 @@
 
   @GraphQLField
   @GraphQLNonNull
-  @GraphQLName(VALUE_TYPE_KEY)
-  ValueType valueType();
+  @GraphQLName(LABEL_APPLICATION_ACTION_TYPE)
+  LabelApplicationActionType labelApplicationActionType();
```

Comment:
We've updated the java class name and the java field name, but the important ones are the ones in the schema, and those weren't changed. The java names and schema field names, like we did elsewhere in this PR, need not be fully qualified - only the types need that.

For the enum, I'd expect:
schema name -> "LabelApplicationActionType"
java (class) name -> "ActionType"

For the field:
java and schema name: "type"

Reply:
Changed.